fairroot/base/sim/FairModule.cxx

```diff
@@ -251,7 +241,7 @@ void FairModule::ProcessNodes(TList* aList)
         if (node->isSensitive() && fActive) {
             volume->setModId(fModId);
             volume->SetModule(this);
-            svList->Add(volume);
+            svList.push_back(volume);
```

Comment:
Follow-up from https://github.com/FairRootGroup/FairRoot/pull/1497#discussion_r1514827979

> So caching `fAllSensitiveVolumes` in a local variable is somewhat a preparation for (3). There it would look like
> 
> ```c++
> auto& svList = FairRunSim::Instance()->fAllSensitiveVolumes;
> ```

But why keep the "acceptable" ([S.2](https://github.com/FairRootGroup/FairRoot/blob/dev/CONTRIBUTING.md#s2-prefer-variable-names-describing-what-they-contain)) variable name ;) Better name would be `sensitiveVolumes`. Or, in this case, just

```suggestion
            fAllSensitiveVolumes.push_back(volume);
```

and later

```suggestion
            FairRunSim::Instance()->fAllSensitiveVolumes.push_back(volume);
```

Reply:
I would like to avoid `FairRunSim::Instance()` inside a loop. But if it's not in a loop (one case is just inside an `if`, I think), you're right.

You're right, the local variable (if I need to touch all references anyway, should be named like you said `allS|s]ensitiveVolumes`.

I'll fix both things.

Reply:
Okay, force pushed a new version. I switched them all over. No caching local variables any more.

I also updated the preview of moving this to a normal member variable of FairRunSim: Top commit of https://github.com/ChristianTackeGSI/FairRoot/tree/pr/svlist_v2